jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -400,15 +406,9 @@ private static String startLocalAgent() throws IbmAttachOperationFailedException
 		} catch (Throwable e) {
 			throw new IbmAttachOperationFailedException("startLocalManagementAgent error starting agent:" + e.getClass() + " " + e.getMessage());		 //$NON-NLS-1$ //$NON-NLS-2$
 		}
-		/*
-		 * sun.management.Agent.startLocalManagementAgent() in Java 8 sets 
-		 * c.s.m.j.localConnectorAddress in System.properties.
-		 * jdk.internal.agent.Agent.startLocalManagementAgent() in Java 9 sets 
-		 * c.s.m.j.localConnectorAddress in a different Properties object.
-		 */
+
 		Properties systemProperties = com.ibm.oti.vm.VM.getVMLangAccess().internalGetProperties();
 		String addr;
-		/*[IF Sidecar19-SE-B165]*/
```

Comment:
Does removing this break the IBM internal builds of Java 8 and Java 9 b148?

Reply:
No.  Personal build (including the TestManagementAgent jobs) ran and passed.